src/spikeinterface/extractors/alfsortingextractor.py

```diff
@@ -13,6 +13,10 @@
     HAVE_PANDAS = True
 except:
     HAVE_PANDAS = False
+try:
+    import one.alf.io as alfio
+except:
+    pass
```

Comment:
Yes but if I put this in the init of the package extractors it will impact all extractors no ?
Are `pandas` and `ONE-api` first citizens of the dependencies ?

Reply:
I mean the init of the class, not of the module

Reply:
can you move this in `ALFSortingExtractor.__init__()` we decided to not use anymore this `try: import `at module level.